apis/server/network_bridge.go

```diff
@@ -77,6 +77,31 @@ func (s *Server) deleteNetwork(ctx context.Context, rw http.ResponseWriter, req
 	return nil
 }
 
+func (s *Server) connectToNetwork(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
+	networkIDOrName := mux.Vars(req)["id"]
+
+	net, err := s.NetworkMgr.Get(ctx, networkIDOrName)
```

Comment:
How about moving this part after request body validation. Maybe the order below will be better:

* validation thing
* inner logic within pouchd's manager

WDYT?